Makefile

```diff
@@ -15,7 +15,7 @@ BUILD_TOOLS_DIR = $(BUILD_DIR)/tools
 GO_ENV_VARS = GO111MODULE=on GOOS=windows
 
 .PHONY: all
-all: generate compile lint
```

Comment:
Is there a reason why we don't want to perform generate ? Wouldn't that result in folks checking in code without running generate ?

Reply:
it does not work without installing protoc etc.  I am not sure whether we can install it easily.

Reply:
I see. Is there a way for us to run specific make targets ?
If so we could run :
```
make compile
make test
```

and get a compile and run the test ?